### DISCUSSION TITLE
Has OCRmyPDF already been run before?

### DISCUSSION THREAD

**Discussion by tcurdt** [ocrmypdf/OCRmyPDF#1371] [Q&A]
Is there a way to tell whether a PDF was already run through OCRmyPDF?
...and/or whether it contains an OCR layer?

**Comment by jbarlow83** [comment]
Ocrmypdf updates the Creator and Producer metadata tags to indicate its version.

**Reply by tcurdt** [reply]
Great. Any suggestion on how to extract/check those (from a shell script)?

**Reply by tcurdt** [reply]
`pdfinfo some.pdf | grep Creator` seems to do it.
